package.json

```diff
@@ -219,6 +219,23 @@
           ],
           "description": "Behavior when adding more memory beyond the current view."
         },
+        "memory-inspector.addressPadding": {
+          "type": "string",
+          "enum": [
+            "Minimal",
+            "Unpadded",
+            "32bit",
+            "64bit"
+          ],
+          "default": "Minimal",
+          "enumDescriptions": [
+            "Minimal padding based on the longest address in range.",
```

Comment:
Just a few suggestions for the wording:
```
Pads address values with leading zeroes to largest address in loaded range.
Disables padding of address values with leading zeroes.
Pads address values with leading zeroes to 32-bit.
Pads address values with leading zeroes to 64-bit.
```
`zeroes` could also be replaced by `0s`. Use should be also aligned with below `description` text.

@colin-grant-work , @gbodeen , @thegecko , any thoughts about this from native English speakers?

Reply:
I think the 'with leading zeroes' part could be a bit redundant, since that's also part of the preference description. Personally, I'd go short, maybe something like 

```jsonc
[
"Pads to shortest uniform length", // This one is the trickiest one to describe clearly. Open to suggestions.
"Disables padding",
"Pads to 32 bits",
"Pads to 64 bits"
]

Reply:
Probably
```
[
"Pads to largest address width in loaded range", // This one is the trickiest one to describe clearly. Open to suggestions.
"Disables padding",
"Pads to 32 bits",
"Pads to 64 bits"
]
```
?